docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -23,6 +23,11 @@ The config file is specified in JSON format. It contains rules that define
 which users have access to which resources. The rules are read from top to bottom
 and the first matching rule is applied. If no rule matches, access is denied.
 
+.. note::
+
+    Access control is enforced on Trino coordinator, therefore access control
```

Comment:
is enforced on the Trino coordinator. Therefore ..

Reply:
you could even leave out "Trino"

Reply:
addressed in https://github.com/trinodb/trino/pull/6826/commits/7e91bc39e6de2051b3c9ca314107b7e4bcb3031e